src/app/auth.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
```

Comment:
are there any issues with doing auth 'client side' - can a malicious client control this? assuming not?

Reply:
Middleware and database level security still protect us from those without authentication. The one thing that concerns me about a malicious client is whether sensitive data remains in cache after an authenticated user has logged out although I'm not sure if that's something we should be concerned with.